e2e/specs/sample-test.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { test, expect } from '@playwright/test';
+import { HomePage } from '../pages';
+
+// This test is a sample E2E test. You can delete it.
+
+test('Sample test', async ({ page }) => {
```

Comment:
Let's switch this up with the landing page for dispensing otherwise this becomes boiler plate